LICENSES.txt

```diff
@@ -170,28 +170,29 @@ Apache-2.0
   mercator_2.13-0.2.1.jar
   metrics-core-3.2.4.jar
   netty-3.10.6.Final.jar
-  netty-all-4.1.77.Final.jar
+  netty-all-4.1.89.Final.jar
   netty-buffer-4.1.93.Final.jar
   netty-codec-4.1.93.Final.jar
-  netty-codec-dns-4.1.77.Final.jar
-  netty-codec-haproxy-4.1.77.Final.jar
+  netty-codec-dns-4.1.89.Final.jar
+  netty-codec-haproxy-4.1.89.Final.jar
   netty-codec-http-4.1.93.Final.jar
-  netty-codec-http2-4.1.77.Final.jar
-  netty-codec-memcache-4.1.77.Final.jar
-  netty-codec-mqtt-4.1.77.Final.jar
-  netty-codec-redis-4.1.77.Final.jar
-  netty-codec-smtp-4.1.77.Final.jar
-  netty-codec-socks-4.1.77.Final.jar
-  netty-codec-stomp-4.1.77.Final.jar
-  netty-codec-xml-4.1.77.Final.jar
+  netty-codec-http2-4.1.89.Final.jar
+  netty-codec-memcache-4.1.89.Final.jar
+  netty-codec-mqtt-4.1.89.Final.jar
+  netty-codec-redis-4.1.89.Final.jar
+  netty-codec-smtp-4.1.89.Final.jar
+  netty-codec-socks-4.1.89.Final.jar
+  netty-codec-stomp-4.1.89.Final.jar
+  netty-codec-xml-4.1.89.Final.jar
   netty-common-4.1.93.Final.jar
   netty-handler-4.1.93.Final.jar
-  netty-handler-proxy-4.1.77.Final.jar
+  netty-handler-proxy-4.1.89.Final.jar
+  netty-handler-ssl-ocsp-4.1.89.Final.jar
   netty-resolver-4.1.93.Final.jar
-  netty-resolver-dns-4.1.77.Final.jar
-  netty-resolver-dns-classes-macos-4.1.77.Final.jar
-  netty-resolver-dns-native-macos-4.1.77.Final-osx-aarch_64.jar
-  netty-resolver-dns-native-macos-4.1.77.Final-osx-x86_64.jar
+  netty-resolver-dns-4.1.89.Final.jar
+  netty-resolver-dns-classes-macos-4.1.89.Final.jar
+  netty-resolver-dns-native-macos-4.1.89.Final-osx-aarch_64.jar
+  netty-resolver-dns-native-macos-4.1.89.Final-osx-x86_64.jar
```

Comment:
Probably there is a sensible reason but I cannot find these 2 dependencies when I list dependencies locally 🤷‍♀️

Reply:
I find: io.netty:netty-resolver-dns-native-macos:4.1.89.Final and am not sure exactly how it works with splitting out jars to -osx etc 🤔

Reply:
Same, I am not to worried about it though so please go ahead and merge